2-Browsers/Week1/assignment/ex4-whatsTheTime/index.js

```diff
@@ -7,7 +7,20 @@ Full description at: https://github.com/HackYourFuture/Assignments/tree/main/2-B
 2. Have the function execute when it's loading in the browser.
 ------------------------------------------------------------------------------*/
 function addCurrentTime() {
-  // TODO complete this function
-}
+  const time = document.createElement('h3');
+  time.classList.add('clock');
+  document.body.appendChild(time);
 
-// TODO execute `addCurrentTime` when the browser has completed loading the page
+  function updateTime() {
+    const date = new Date();
+    const hours = date.getHours();
+    const minutes = date.getMinutes();
+    const seconds = date.getSeconds();
+    const currentTime = `${hours >= 10 ? hours : '0' + hours}:${minutes >= 10 ? minutes : '0' + minutes}:${seconds >= 10 ? seconds : '0' + seconds}${hours > 12 ? 'PM' : 'AM'}`;
```

Comment:
What you've done works, but I would have rather seen you use `padStart` just like I recommended, since it'd give a cleaner and clearer code:

```suggestion
    const date = new Date();
    const hours = date.getHours().toString().padStart(2, '0');
    const minutes = date.getMinutes().toString().padStart(2, '0');
    const seconds = date.getSeconds().toString().padStart(2, '0');
    const currentTime = `${hours}:${minutes}:${seconds}${hours > 12 ? 'PM' : 'AM'}`;
```